modules/developers-guide/pages/concepts/canisters-code.adoc

```diff
@@ -34,30 +34,30 @@ This concept of a canister consisting of both program and state is an important
 a| Allow the user to query the current state of a canister or call a function that operates on the canister’s state *without changing it*.
 
 * Are synchronous and answered immediately.
-* Can be made to any node that holds the canister and do not require consensus to verify the result. 
-There is an inherent tradeoff between security and performance because the reply from a single node might be untrustworthy or inaccurate.
-* Do not allow changes to the state of the canister to be persisted. Essentially, programs use query calls to perform read-only operations.
+* Can be made to any node that holds the canister; the result does not go through consensus.
+That is, there is an inherent tradeoff between security and performance: the reply from a single node is fast, but might be untrustworthy or inaccurate.
+* Do not allow changes to the state of the canister to be persisted, so essentially query calls are read-only operations.
 * Do not allow the called canister to invoke functions exposed by other canisters as inter-canister calls. (Note that this restriction is temporary and that canisters will be able to invoke functions exposed by other canisters when processing query calls in the future.)
 
 | Update calls
 a| Allow the user to change the state of the canister and have *changes persisted*.
 
 * Are answered asynchronously.
 * Must pass through consensus to return the result. 
-Because consensus is required, changing the state of a canister can take time. Therefore, update calls use the actor-based programming model (with state isolation) to allow concurrent and asynchronous processing. 
-There is an inherent tradeoff between security and performance because two-thirds of the replicas in a subnet must agree on the result.
+Because consensus is required, changing the state of a canister, and returning the result can take time.
+There is an inherent tradeoff between security and performance: the result is trustworthy because two-thirds of the replicas in a subnet must agree on the result, but the call is slow.
```

Comment:
Should we refer to certified variables here?

Reply:
Not really: the emphasis is on the difference between update and query calls and mentioning certified variables here I think would be distracting from the main message